### PR TITLE
Fix copy on screen 3 & 6

### PR DESCRIPTION
**Pull request by smn**

### CODE REVIEW

**Comment by smn** [comment]
@KaitCrawford ready for review.

File: test/app.test.js [hunk 1]
```diff
@@ -72,6 +72,19 @@ describe("app", function() {
...
+                        .setup.user.state('Facility_Code_Confirm')
+                        .input('1')
+                        .check.interaction({
+                            state: 'MSISDN_Entry',
+                            reply: /Please enter the South African cell phone number/
+                        })
+                        .run();
+            });
+        });
+
```
Comment (KaitCrawford) [inline test/app.test.js:87]: I'm unfamiliar with this sort of testing so will have to educate myself before I can review it.

**Comment by KaitCrawford** [comment]
:+1: besides the testing stuff that I'm unfamiliar with

File: test/app.test.js (hunk 1 above)
Comment (smn) [inline test/app.test.js:87]: Yes :) Which is why I'm asking you review it :) It's the best way to learn.

Vumi Go's jsbox essentially allows people to design state machines. Each screen is a state and when certain conditions are met it transitions to the next state, which is a new screen (which may end up being USSD, SMS, Twitter tweet etc ...)

Comment (smn) [inline test/app.test.js:87]: Oops hit enter too early.

In the test we're priming the state, we tell it what state it is in (`Facility_Code_Confirm`) and then provide it some inputs (`1` in this case) and then verify that it results in the state transition to the next named state (`MSISDN_Entry`) and that the content returned from that next state matches the regex provided.

File: go-app.js [hunk 2]
```diff
@@ -150,7 +152,7 @@ go.app = function() {
 
     self.states.add('Patient_Abroad_Entry', function(name) {
       return new ChoiceState(name, {
-        question: $("Has the patient travelled outside of the country in the past 2 weeks:"),
+        question: $("Has the patient travelled outside of the country in the past 3 weeks:"),
         choices: [
           new Choice('1', $("Yes")),
           new Choice('2', $("No")),
```
Comment (smn) [inline go-app.js:158]: This `go-app.js` file is compiled by grunt, a command line utility that compiles all the different JS files into 1 big one that we can then upload in the UI and have Vumi Go run. When reviewing you only need to look at the files changed under `src/*` as those are compiled into this one.

**Pull request merged**